pyro/distributions/multivariate_normal.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+import torch
+from torch.autograd import Variable
+
+from pyro.distributions.distribution import Distribution
+
+
+def potri_compat(var):
+    return Variable(torch.potri(var.data)) if torch.__version__ < '0.3.0' else torch.potri(var)
+
+
+class MultivariateNormal(Distribution):
```

Comment:
nit: You can decorate with `@copy_docs_from(Distribution)` and then omit trivial method docstrings. See other distributions for usage.